packages/server/graphql/mutations/helpers/generateWholeMeetingSentimentScore.ts

```diff
@@ -2,14 +2,9 @@ import {DataLoaderWorker} from '../../graphql'
 
 const generateWholeMeetingSentimentScore = async (
   meetingId: string,
-  facilitatorUserId: string,
   dataLoader: DataLoaderWorker
 ) => {
-  const [facilitator, reflections] = await Promise.all([
-    dataLoader.get('users').loadNonNull(facilitatorUserId),
-    dataLoader.get('retroReflectionsByMeetingId').load(meetingId)
-  ])
-  if (!facilitator.featureFlags.includes('aiSummary')) return undefined
```

Comment:
I removed this check with the thinking:
1. It's not expensive at all to calculate an average score
2. We are just writing that score to DB. If there would be any limit in the future, it'd be on the front-end where we decide when we can surface this data.

Reply:
@tianrunhe my concern is some users don't want their data to be used by AI, so I think it would be safer to leave the feature flag check in here. 

Ideally, we roll out the AI Summary feature soon and then invert the feature flag, so we would just have a small exempt list of users/orgs

Reply:
Fair enough!